### DISCUSSION TITLE
I wonder if there will be a Union file system like hugo after 11ty? | 不知道11ty以后会有类似hugo的Union file system吗？

### DISCUSSION THREAD

**Discussion by rebron1900** [11ty/eleventy#4097] [Enhancement Queue]
### Is your feature request related to a problem? Please describe.

在使用一些模板的时候，如果我对模板进行了大量的修改后，如果该模板作者进行了更新，我如果要同步这个更新会非常麻烦。

When using some templates, if I make a lot of changes to a template, if the author of that template makes an update, it would be very cumbersome for me to synchronize that update.

### Describe the solution you'd like

希望就像hugo的Union file system那样运作。

**Comment by zachleat** [comment]
Hugo docs link: https://gohugo.io/getting-started/directory-structure/#union-file-system

**Comment by zachleat** [comment]
> By default this repo closes enhancement request issues and places them in a queue for folks to upvote. We do this to avoid a large build-up of open issues. We got this idea from the [lodash project](https://twitter.com/samselikoff/status/991395669016436736).

> Don’t forget to upvote the first comment with 👍 to register your vote! You can view [the current enhancement backlog](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+).

**Comment by zachleat** [comment]
Related to https://github.com/11ty/eleventy/discussions/3996
